$NetBSD: patch-af,v 1.7 1999/12/12 19:06:37 jdolecek Exp $

Obsoleted, the file is no longer pertinent in KDE2       jdolecek 1999-12-12
--- ./kdehelp/khelp.h	Mon Mar 30 21:28:15 1998
+++ /usr/pkgsrc/x11/kdebase/work.i386.unpatched/kdebase-1.1.2/kdehelp/khelp.h	Fri Nov 12 05:10:33 1999
@@ -22,9 +22,15 @@
 #include <sys/msg.h>
 
 #ifndef __osf__
-#ifdef BSD
-#define msgbuf mymsg
-#endif
+# ifdef BSD
+#  define msgbuf mymsg
+#  if __NetBSD_Version__ >= 104110000	/* 1.4K or higher */
+struct mymsg {
+	long	mtype;		/* message type (+ve integer) */
+	char	mtext[1];	/* message body */
+};      
+#  endif
+# endif
 #endif
 
 class KHelpMsg : protected msgbuf
